Web/src/components/CustomTablePagination.jsx

```
@@ -0,0 +1,158 @@
+
+
+    const [isCustomDialogOpen, setIsCustomDialogOpen] = useState(false);
+    const [customRowsPerPage, setCustomRowsPerPage] = useState('');
+
+
+    // Handler for custom dialog confirmation
+    const handleCustomDialogConfirm = () => {
+        // Validate input
+        const parsedValue = parseInt(customRowsPerPage, 10);
+        
+        if (!isNaN(parsedValue) && parsedValue > 0) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)